src/lib/types/chat.ts

```
@@ -63,7 +64,7 @@ export type ChatState = {
     isDeleted: (eventId: string) => boolean;
     getMessageReactionsSummary: (messageId: string) => ReactionSummary[];
     hasReactions: (message: Message) => boolean;
-    sendReaction: (group: any, reaction: string, messageId: string) => Promise<NEvent | null>;
+    clickReaction: (group: any, reaction: string, messageId: string) => Promise<NEvent | null>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)